script/Ciel/Day 7/07_01_CIEL07_4.txt

```diff
@@ -259,28 +279,30 @@ That's all, apparently.
 [sha:79e6f7177a785a0b7d49402e4fba9f21e0a8ffd5]{
 -- Page 420, Offset 15812.
 -- アルクェイドが言っていたのはその水路の事だろうか……？
-I wonder if Arcueid was talking about this waterway?
+I wonder if Arcueid was talking about this aqueduct...?
 }
 [sha:f13f16c0d419a5018ca7e3d2c6f7420e9bd962e0]{
 -- Page 421, Offset 15813.
 -- 「そう。お父様は屋敷の造りに手を入れたがらなかったから、そういう事もあるのでしょうね。いい機会だし、今の案件が落ち着いたら業者を入れて様子を見ましょうか」
-"Quite. Father never held back in modifying the construction of the mansion, so something like that is possible.
- As good an opportunity as any――once my current matter has settled, I'll pay for a contractor to take a look."
+"Good point.
+ Father was very reluctant to make any changes to the construction of the mansion, so something along those lines is possible.
+ Well, this is as good an excuse as any――once my agenda has cleared out a little, I'll hire a contractor to take a look."
 }
 [sha:1a562d8ccf4fcfd79617e8bcdb8ff4f452000b1f]{
 -- Page 422, Offset 15814.
 -- 水路の話は終わり、何でもない会話に戻る。
-The subject of the waterway comes to an end, and we return to talking about nothing in particular again.
+The subject of the aqueduct comes to an end, and we return to talking about nothing in particular again.
 }
 [sha:b88104916e3ad737ac6d8cce8749a2e5210796a6]{
 -- Page 422, Offset 15815.
 -- 結局お茶会は８時過ぎまで続いたが、俺が入浴する事でお開きになった。
-Ultimately, the tea party goes on until eight o'clock, with me taking my leave on account of the bath.
+The tea party ends up carrying on past eight o'clock,
+ though I have to excuse myself to go take a bath.
```

Comment:
> The tea party ends up carrying on past eight o'clock, though I **had** to excuse myself to go take a bath.

Sounds to me like the 'taking bath' is a past event that he's commenting on, while the 'tea party' is supposedly still ongoing.

Reply:
if that's the case (and i can certainly see why it would be), it would be "ended up" on top of "had".